src/main/java/org/kairosdb/core/http/rest/MetricsResource.java

```diff
@@ -438,7 +438,13 @@ public Response get(@Context HttpHeaders httpHeaders, String json) throws Except
 
 						DatastoreQuery dq = datastore.createQuery(query);
```

Comment:
What's involved in creating an instance of this class?  Can/should we avoid doing that if we know the query is rejected?